examples/quick-start-java/README.md

```diff
@@ -0,0 +1,10 @@
+# Quick Start - Ruby on Rails app with OpenTelemetry and Tracetest
+
+> [Read the detailed recipe for setting up OpenTelemetry Collector with Tractest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
+This is a simple quick start on how to configure a Ruby app (with [Rails](https://rubyonrails.org/) framework) to use OpenTelemetry instrumentation with traces, and Tracetest for enhancing your e2e and integration tests with trace-based testing.
```

Comment:
I know you like Rails, but I think this should be about Java :rofl:

Reply:
🙃 
(Thanks for this catch!)